CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 34 Release Notes
 
+### Query API
+
+* Fixes a bug which stopped `includeSubworkflow=false` from paging correctly and subworkflows from being discounted correctly from `totalResultsCount`.
```

Comment:
ToL: Either don't mention paging yet (as it's still unsupported!), or finally euthanize the bad RFC 5988 pagination headers and add pagination to cromwell.yaml, /docs, etc.

Reply:
We're going to need to [modify our pagination scheme](https://github.com/ga4gh/workflow-execution-service-schemas/blob/develop/openapi/workflow_execution_service.swagger.yaml#L780) soon anyways, FWIW

Reply:
I've removed the bad headers and made page and pageSize public because it's useful, we rely on it in JMUI, and we don't want it to get "nobody uses this"'d.

If it's wrong, that's cause for a ticket, not a "hide it under the rug" IMO and whatever query does end up looking like, it needs to include pagination.